src/perl/Permabit/UserModule.pm

```diff
@@ -51,6 +51,14 @@ sub loadFromBinaryRPM {
   my $supportName = "$modFileName-support";
   my $supportFile = ($self->{useDistribution}) ? makeFullPath($self->{modDir}, "$supportName-*.rpm")
                                                : makeFullPath($topdir, "$supportName-*.rpm");
+   # if useDnf(useLinuxNextModule), download vdo package from distribution repo
+   if ($self->{useDnf}) {
+    my $getFromDnf = join(' ',
+                          "dnf", "download", "--destdir", "$topdir",
+		          "$modFileName-support"
+                          );
```

Comment:
$modFileName-support is alredy defined as $supprotName, just use that. Alos, don't have ); on a line by itself for a function call.

Reply:
Here also, format the join call like a normal function.